client/js/components/NavigationDrawer.vue

```diff
@@ -65,7 +65,7 @@ export default {
             ...rules,
           ]
         : [
-            { text: 'Logout', icon: 'logout', page: { name: 'Login' } },
+            { text: 'Logout', icon: 'logout', page: { name: 'Logout' } },
```

Comment:
This prefers middleware to handle the logout process which feels much more reliable and safe.